specification/communication/data-plane/CallAutomation/stable/2023-03-06/communicationservicescallautomation.json

```diff
@@ -0,0 +1,2285 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "version": "2023-03-06",
+    "title": "Azure Communication Service Call Automation APIs",
+    "description": "Azure Communication Service Call Automation APIs"
+  },
+  "schemes": [
+    "https"
+  ],
+  "consumes": [
+    "application/json"
+  ],
+  "produces": [
+    "application/json"
+  ],
+  "paths": {
+    "/calling/callConnections": {
+      "post": {
+        "tags": [
+          "PreConnections"
+        ],
+        "summary": "Create an outbound call.",
+        "description": "Create an outbound call.",
+        "operationId": "CreateCall",
```

Comment:
Some operationId is noun_verb e.g. CallConnection_GetCall, some not e.g. this one, may consider be consistent.

Reply:
This was intentionally like this because how current pipeline builds API spec reference automatically. It uses operationId to group operations, and we wanted to have these named in specific way to fit our needs.